packages/enhanced/src/lib/container/ContainerPlugin.ts

```diff
@@ -193,16 +194,38 @@ class ContainerPlugin {
         {
           name,
           filename,
-          runtime,
+          runtime: hasSingleRuntimeChunk ? false : runtime,
```

Comment:
if it has custom runtime chunk rules, then emit a standalone remoteEntry by opting out with runtime: false